src/commcare_cloud/commands/ansible/service.py

```diff
@@ -457,6 +457,18 @@ class CommCare(SingleSupervisorService):
     def supervisor_process_name(self):
         return 'all'
 
+    def run(self, action, host_pattern=None, process_pattern=None):
+        if action == 'restart':
+            puts(color_warning("The 'commcare' service command rejects the 'restart' action"))
+            puts(color_warning("in order to protect against accidental downtime."))
+            puts(color_notice("For a no-downtime restart of commcare-hq services, please use one of the following"))
+            puts(color_code(f"  cchq {self.environment.name} fab restart_services  # for all processes that run commcare-hq code."))
+            puts(color_code(f"  cchq {self.environment.name} fab restart_webworkers  # for webworkers only"))
+            puts(color_notice("To restart formplayer, which always causes downtime, use the 'formplayer' service command."))
```

Comment:
It seems better to be explicit about the command rather than leaving room to guess what _'formplayer' service command_ means.
```suggestion
            puts(color_code(f"  cchq {self.environment.name} service formplayer restart  # for formplayer, causes downtime"))
```

Reply:
What I wanted to avoid is to make it too easy to copy-paste a command that one should avoid running because it causes downtime. So I intentionally inlined the ones above and then referenced this one verbally. It's a hedge against people not reading carefully.